control-plane-operator/controllers/hostedcontrolplane/manifests/pki.go

```diff
@@ -32,6 +32,15 @@ func CombinedCAConfigMap(ns string) *corev1.ConfigMap {
 	}
 }
 
+func AggregateClientCAConfigMap(ns string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "aggregator-client-ca",
```

Comment:
this needs to be wired into the KAS config

Reply:
I missed https://github.com/openshift/hypershift/pull/1831/files#diff-2ba4d70b5e57d9efdf316eca3d90a1ec65c00aefaab7c3984937351bf2f3ff77R558, this should work